functions.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Creates a connection to the collectorapp database
+ *
+ * @return PDO
+ */
+    function createDBConnection(): PDO {
```

Comment:
not sure if you were told to put this in a function but would this just be better to run out of the function?

Reply:
Was told to create it so I can use it again later :)